bindings/python/examples/dnn/dnn.py

```diff
@@ -39,7 +39,7 @@
 
 ip = "10.43.0.1" # Set to "10.43.0.1" if networking is used.
 config = "config/config_adsd3500_adsd3100.json"
-mode = "sr-qnative"
+mode = 3
```

Comment:
isn't sr-qnative supposed to be index 2? Or it doesn't matter for this example?

Reply:
It doesn't really matter.